### PR TITLE
Added ZEIT topic

### PR DESCRIPTION
**Pull request by leo**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

ZEIT provides a variety of widely used open source tools and a platform for globally distributed single-command deployments ([Now](https://zeit.co/now)). Since many people in our community are building open source plugins/packages to extend our software, it would be wonderful to bundle them all in a GitHub Explore Topic to make them easier to find.

### CODE REVIEW

**Comment by leo** [comment]
@Haroenv Fixed!

**Comment by Haroenv** [comment]
oh oops, I misread the CI failure, it's actually 288x288 @leo

**Comment by leo** [comment]
Fixed again!

File: topics/zeit/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+display_name: ZEIT
+github_url: https://github.com/zeit
+logo: zeit.png
+short_description: Our mission is to make cloud computing as easy and accessible as mobile computing.
```
Comment (nayafia) [inline topics/zeit/index.md]: Suggest using more descriptive, wiki-style language here, e.g.

> Zeit is a company that builds tools to make cloud computing easy

File: topics/zeit/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+topic: zeit
+url: https://zeit.co
+---
+Our mission is to make cloud computing as easy and accessible as mobile computing.
```
Comment (nayafia) [inline topics/zeit/index.md]: Ditto prev comment; a couple sentences about what Zeit is, using a more neutral POV would be better!

File: topics/zeit/index.md (hunk 1 above)
Comment (leo) [reply topics/zeit/index.md]: @nayafia ✅

File: topics/zeit/index.md (hunk 2 above)
Comment (leo) [reply topics/zeit/index.md]: @nayafia ✅

**Comment by nayafia** [comment]
Made a couple lil grammar tweaks to better match our style guide. Merging! :tada:

**Comment by leo** [comment]
@nayafia Thank you! When are you deploying this?

**Comment by nayafia** [comment]
@leo sorry, forgot to respond here - it's live: https://github.com/topics/zeit

**Pull request merged**